sa/proto/sa.proto

```diff
@@ -253,12 +255,13 @@ message OrderRequest {
 }
 
 message NewOrderRequest {
-  // Next unused field number: 6
+  // Next unused field number: 7
   int64 registrationID = 1;
   reserved 2; // Previously expiresNS
   google.protobuf.Timestamp expires = 5;
   repeated string names = 3;
   repeated int64 v2Authorizations = 4;
+  string replaces = 6;
```

Comment:
nit: I'd maybe make this "replacesSerial", both to make it clear what the contents of the field are, and because it contains different content than the `replaces` field at the ACME API layer (where it also contains an issuer identifier).

Reply:
Agreed!